### DISCUSSION TITLE
May we upgrade from 5.1.33.64 to 2022.1

### DISCUSSION THREAD

**Discussion by ruslan15sof** [datalust/seq-tickets#1698] [Help]
We’ve upgraded to 5.1.33.64, but we didn’t receive any pop-up message that migration is completed, 3 days passed since then.
May we upgrade to the next version 2022.1?

**Comment by nblumhardt** [comment]
Hi Ruslan! Thank you for reaching out. I have just noticed that my instructions for you were partially incorrect - you won't receive a migration completion message when upgrading to 5.1; this will actually occur after upgrading to 2022.1.

The time spent on 5.1 is intended to ensure that your most recent extent is using the new storage engine, so that you can immediately access this data after performing the next migration step (which will migrate other extents in the background).

To check, if the most recent folder in your `C:\ProgramData\Seq\Extents` directory contains an `extent.flare` file then you're all done and you can continue upgrading.

Best regards,
Nick

**Comment by ruslan15sof** [comment]
Thank you!

From: Nicholas Blumhardt ***@***.***>
Sent: Monday, November 28, 2022 7:03 PM
To: datalust/seq-tickets ***@***.***>
Cc: Ruslan Sabitov ***@***.***>; Author ***@***.***>
Subject: [External] Re: [datalust/seq-tickets] May we upgrade from 5.1.33.64 to 2022.1 (Discussion #1698)

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Hi Ruslan! Thank you for reaching out. I have just noticed that my instructions for you were partially incorrect - you won't receive a migration completion message when upgrading to 5.1; this will actually occur after upgrading to 2022.1.

The time spent on 5.1 is intended to ensure that your most recent extent is using the new storage engine, so that you can immediately access this data after performing the next migration step (which will migrate other extents in the background).

To check, if the most recent folder in your C:\ProgramData\Seq\Extents directory contains an extent.flare file then you're all done and you can continue upgrading.

Best regards,
Nick
